src/agent/onefuzz-agent/src/tasks/merge/libfuzzer_merge.rs

```diff
@@ -113,6 +101,41 @@ async fn process_message(config: Arc<Config>) -> Result<()> {
     }
 }
 
+async fn sync_and_merge(
+    config: Arc<Config>,
+    input_dirs: Vec<impl AsRef<Path>>,
+) -> Result<LibFuzzerMergeOutput> {
+    config.unique_inputs.sync_pull().await?;
+    match merge_inputs(config.clone(), input_dirs).await {
```

Comment:
Does this support deleting files?

A common use case for merging is to reduce the set of inputs that provide the minimal number of files with the maximal coverage.